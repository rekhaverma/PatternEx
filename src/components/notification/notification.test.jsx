import React from 'react';
import Notification from './notification.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<Notification />', () => {
  it('should match with snapshot', () => {
    const wrapper = shallow(<Notification />);
    expect(wrapper).toMatchSnapshot();
  });
});
