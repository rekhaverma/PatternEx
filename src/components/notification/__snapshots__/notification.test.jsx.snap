// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notification /> should match with snapshot 1`] = `
<div
  className="notification notification--info"
>
  <span
    className="notification__icon icon-info"
  />
  <div>
    <FormattedMessage
      id="notification.text"
      key="0"
      values={
        Object {
          "line": "Updated",
        }
      }
    />
  </div>
  <span
    className="notification__close icon-remove"
    onClick={[Function]}
  />
</div>
`;
